collect_app/src/androidTest/java/org/odk/collect/android/utilities/BitmapScaledToDisplayTest.java

```diff
@@ -25,7 +25,7 @@ public void scaleDownBitmapWhenNeeded() {
         runScaleTest(1000,   1000,    500,    500,    500,    500,    false);
         runScaleTest( 600,    800,    600,    200,    150,    200,    false);
         runScaleTest( 500,    400,    250,    200,    250,    200,    false);
-        runScaleTest(2000,    800,    300,    400,    500,    200,    false);
+        runScaleTest(2000,    800,    300,    400,    333,    133,    false);
```

Comment:
Why did this change cause the test to pass and why was it made? Would be nice if the commit message gave these details :smiley:

Reply:
I am still not sure why this was failing and why the fix was required. But I will surely try to understand and then update it here